### PR TITLE
Switch from Travis-CI to Github Actions

### PR DESCRIPTION
**Pull request by regadas**
Hi @alexarchambault a little bit more verbose compared to `travis` but it gets the job done.

* `build` workflow only runs on branches and PR's

* `publish` workflow is triggered only when a `tag` is pushed; updates the readme and runs `ci-release`

You will need to setup the same secrets as in Travis.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        with:
+          java-version: "8"
+      - run: .github/scripts/ci-setup.sh
+      - run: ~/.bin/amm .github/scripts/generate-readme.sc
+      - uses: stefanzweifel/git-auto-commit-action@v4.1.6
+        with:
+          file_pattern: README.md
+          commit_message: Update README.md
+          branch: master
+          commit_author: Github Actions <actions@github.com>
```
Comment (regadas) [inline .github/workflows/publish.yml:43]: Let me know if you want to be the author here.

Comment (alexarchambault) [reply .github/workflows/publish.yml:43]: `Github Actions <actions@github.com>` is better 👍

**Comment by regadas** [comment]
@alexarchambault Aaha no problem! so this kinda worked eheh 😄 saw this https://github.com/coursier/apps/commit/9467f3faeaddee2206d71bb967c4267b313093e5 yeah kinda makes sense i can publish fails 👍

**Comment by alexarchambault** [comment]
Yes, seems [it worked](https://repo1.maven.org/maven2/io/get-coursier/apps/0.0.22)! No worry, the problem was because of the commit created by the README update. sbt-dyn (pulled by sbt-ci-release) was setting the version to something like `0.0.22+1-…-SNAPSHOT` because of that, which was confusing sbt-ci-release. Pulling off that kind of CI pipeline on first try would be quite a feat actually ^^

**Pull request merged**